src/models/thelio-mira-r1.0/README.md

```diff
@@ -0,0 +1,81 @@
+# Thelio Mira (thelio-mira-r1.0)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Thelio Mira](./img/thelio-mira-r1.0.png)
+
+The System76 Thelio Mira is a desktop with the following specifications:
+
+- CPU options
+    - Supports AMD 1st through 4th gen Ryzen AM4 socket processors
+    - Tested with the following processors:
+        - [Ryzen 5 3600](https://www.amd.com/en/products/cpu/amd-ryzen-5-3600)
+        - [Ryzen 5 5600X](https://www.amd.com/en/products/cpu/amd-ryzen-5-5600x)
+        - [Ryzen 7 5800X](https://www.amd.com/en/products/cpu/amd-ryzen-7-5800x)
+        - [Ryzen 9 5900X](https://www.amd.com/en/products/cpu/amd-ryzen-9-5900x)
+        - [Ryzen 9 5950X](https://www.amd.com/en/products/cpu/amd-ryzen-9-5950x)
+- Motherboard
+    - [ASUS ROG Strix X570-E Gaming](https://rog.asus.com/us/motherboards/rog-strix/rog-strix-x570-e-gaming-model/) running System76 Firmware (non-open)
+    - AMD X570 chipset
+- Daughterboard
+    - [Thelio Io](https://github.com/system76/thelio-io) board running [open-source firmware](https://github.com/system76/thelio-io-firmware)
+- GPU options
+    - One or two GPUs, depending on configuration and power requirements (3x PCIe 3.0/4.0 x16 slots)
```

Comment:
Is there a reason we're listing the slots as `3.0/4.0`? The 4.0 spec is backwards-compatible with 3.0, so we should only have to list 4.0, correct? (Mega just calls them 4.0.)

Reply:
Only because I've seen questions about backwards compatibility before. Just saying 4.0 should be enough though, so we can change it if you think it looks better.

Reply:
Looks like it's to distinguish that some of the slots will run at 3.0 with older-generation processors? The motherboard manual only lists the top two as 3.0/4.0, and lists the bottom as 4.0-only (indicating it will run at 4.0 even with 1st gen processors.) See pages 1-6 and 1-7. https://dlcdnets.asus.com/pub/ASUS/mb/SocketAM4/ROG_STRIX_X570-E_GAMING/E15826_ROG_STRIX_X570-E_GAMING_UM_v2_WEB.pdf

Reply:
Looks to me like the manual lists all three x16 slots as 3.0/4.0 and the x1 slots as 4.0 (which _should_ still be backwards compatible because 4.0 only x1 slots makes no sense). I'm leaning towards changing it to `3x PCIe 4.0 x16 slots` for the sake of brevity.